src/main/java/bio/terra/cli/command/cromwell/generate.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Not sure the correct place for this script file. I have 3 proposal for this file location. 1 is under the Docker folder, 2 is under tools folder. 3 is here. I choose 3, because learn from WSM also have similar styles put the script file in the same file with the file who called it. (example: https://github.com/DataBiosphere/terra-workspace-manager/tree/main/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook)